README.md

```diff
@@ -351,8 +351,8 @@ The default configuraton is:
 config.yml:
 ```yaml
 chaplean_api_client:
-    enable_database_logging: false
-    enable_email_logging: false
+    enable_database_logging: false # You can define a list of names of your API (ex: ['foo_api'], ['!foo_api'])
```

Comment:
Maybe to be more specific : "You can define a list of APIs names using chaplean/api-client-bundle (ex: ['foo_api'], ['!foo_api'])"

Reply:
We should be even more explicit and put a block of text explaining the different values allowed and their meaning. Look at the doc for `codes_listened` just bellow.

It may be clearer with something along the lines of:
```yaml
chaplean_api_client:
    # Specify when to log api requests. You can give a boolean to enable or disable globally
    # or give a white list of clients where logging is only enabled for the listed clients (ex: ['foo_api', 'bar_api'])
    # or a black list where logging is enabled for all clients excepted those listed (ex: ['!foo_api', '!bar_api']).
    enable_database_logging: false
    enable_email_logging: false
```